portingdb/htmlreport.py

```diff
@@ -479,34 +480,29 @@ def howto():
     db = current_app.config['DB']()
     query = queries.packages(db)
 
-    # Count the idle and blocked packages
-    idle_query, query = queries.split(query, tables.Package.status == 'idle')
-    idle_len = idle_query.count()
-    blocked_query, query = queries.split(query, tables.Package.status == 'blocked')
+    # Count the blocked packages
+    blocked_query = query.filter(tables.Package.status == 'blocked')
     blocked_len = blocked_query.count()
```

Comment:
If there was a `blocked`, then it wouldn't be better. Here the count method it generates a `SELECT COUNT()` query without getting the actual list of objects.